.github/workflows/ci.yml

```diff
@@ -75,6 +75,13 @@ jobs:
     - name: Credo
       run: mix credo
 
+    - name: Create Dialyzer PLTs
+      if: steps.plt_cache.outputs.cache-hit != 'true'
```

Comment:
There should be an extra `actions/cache` step to cache the PLT file too so this doesn't miss all of the time

Reply:
Yeah, hard to fully test it when the pipeline does not run 😭